Add blog post on harness-first agentic SDLC (#4142)
opensearch-project/project-website: _community_members/ghiyas.md

---
short_name: 'ghiyas'
name: 'Sudeep Ghiya'
photo: '/assets/media/community/members/ghiyas.jpg'
job_title_and_company: 'Software Engineer at AWS'
primary_title: 'Sudeep Ghiya'
title: 'OpenSearch Community Member: Sudeep Ghiya'
breadcrumbs:
  icon: community
  items:
    - title: Community
      url: /community/index.html
    - title: Members
      url: /community/members/index.html
    - title: "Sudeep Ghiya&apos;s Profile"
      url: '/community/members/sudeep-ghiya.html'
permalink: '/community/members/sudeep-ghiya.html'
personas:
  - author
redirect_from: '/authors/ghiyas/'
---

> Sudeep has extensive experience building and scaling control planes. He is a software engineer who is currently leveraging his experience to help build the next version of AWS OpenSearch Serverless using Gen AI tools.

Reviewer: ```suggestion
Sudeep is a software engineer with extensive experience building and scaling control planes. He is currently leveraging his experience to build the next version of AWS OpenSearch Serverless using generative AI tools.
```
